library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss

```diff
@@ -152,4 +169,27 @@
 
 .mat-card-footer {
   padding-bottom: 22px;
+  margin-left: -18px;
+}
+
+.mat-card-sub-title-marign
+{
+  margin-left: -18%;
```

Comment:
Why is the need for minus margins. can we avoid this and fix

Reply:
I tried to remove it.. but to align it i need to add this margin left . i didn't overwrite the main class as you suggested.. i have created a custom class for that